benchmarks/bench_minimum_spanning_tree.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+import torch
+
+import beignet
+
+
+class TimeMinimumSpanningTree:
```

Comment:
Look at the other benchmarks, there should only be one class:

`BenchMinimumSpanningTree`

It should have both `time_minimum_spanning_tree` and `peak_memory_minimum_spanning_tree` methods.